test/google_scraping_web/api/v1/auth_controller_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule GoogleScrapingWeb.Api.V1.AuthControllerTest do
+  use GoogleScrapingWeb.ConnCase, async: true
+
+  describe "POST create/2" do
+    test "given valid user credentials, returns access token", %{conn: conn} do
+      email = "test@test.com"
+      insert(:user, email: email)
+
+      conn =
+        post(conn, Routes.api_auth_path(conn, :create), %{
+          email: email,
+          password: valid_user_password()
+        })
+
+      assert %{
+               "data" => %{
+                 "attributes" => %{
+                   "email" => ^email,
+                   "token" => _
+                 },
+                 "id" => _,
+                 "relationships" => %{},
+                 "type" => "auth"
+               },
+               "included" => []
+             } = json_response(conn, 200)
```

Comment:
Same with the test below

Reply:
For test below it was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/54/commits/8a71099139d2f734a400de9234b4269d5507f0a6